src/base/pages/Landing.vue

```diff
@@ -112,6 +112,21 @@
           href="https://github.com/yunity/karrot-frontend"
         >GitHub</a>
       </i18n>
+
+      <p class="text-center q-mt-lg">
+        <QBtn
+          @click="toggleAbout"
+        >
```

Comment:
The button doesn't look quite fitting in style to me. The simplest change I think it just to add `unelevated` attribute. Still looks a bit different from the other buttons, but I think it's OK.

![aboutbutton](https://user-images.githubusercontent.com/31616/112768080-15885780-9012-11eb-957d-da33721d843b.png)

Reply:
I added unelevated. For some reason I like it being different :) But I don't mind changing to fit the style
This button will be temporary anyhow if we go on to create the about page and improving the landing page